core-blocks/gallery/editor.scss

```diff
@@ -77,6 +77,16 @@
 			border: 1px solid #999;
 		}
 	}
+
+	.editor-rich-text .editor-rich-text__tinymce {
+		a {
+			color: $white;
+		}
+
+		&:focus a[data-mce-selected] {
+			color: rgba(0, 0, 0, 0.2);
```

Comment:
@tofumatt I was just about to edit the commit and improve the formatting 😅 Thank you so much ❤️